.github/workflows/dotnet-sample-app-s3-deploy.yml

```diff
@@ -42,13 +46,68 @@ jobs:
           role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/${{ secrets.E2E_TEST_ROLE_ARN }}
           aws-region: ${{ matrix.aws-region }}
 
+      - name: Update .NET version if not default
+        working-directory: sample-apps/dotnet
+        run: |
+          if [ "${DOTNET_VERSION}" != "${DEFAULT_VERSION}" ]; then
+            echo "Updating .NET version from ${DEFAULT_VERSION} to ${DOTNET_VERSION}"
+            
+            # Function to update version in files
+            update_version() {
+              local file=$1
+              if [ ! -f "$file" ]; then
+                echo "Error: File $file not found"
+                exit 1
+              fi
+              
+              # Update TargetFramework in csproj files
+              if [[ $file == *.csproj ]]; then
+                sed -i "s/<TargetFramework>netcoreapp${DEFAULT_VERSION}<\/TargetFramework>/<TargetFramework>netcoreapp${DOTNET_VERSION}<\/TargetFramework>/g" "$file"
+              # Update version in Dockerfiles
+              elif [[ $file == */Dockerfile ]]; then
```

Comment:
I don't believe ec2 case uses the Dockerfile. We can remove this condition and the content inside it

Reply:
It will be useful for the EKS case though if we end up implementing it

Reply:
Hey, Harry should I retain the Dockerfile changes or should I just delete it?

Reply:
Delete it from this PR

Reply:
Only the docker part right?

Reply:
Remove the condition here too, it doesn't make sense to have it here

Reply:
Okay, I will remove the docker part